service_info_cms/cms_menus.py

```diff
@@ -25,4 +25,23 @@ def modify(self, request, nodes, namespace, root_id, post_cut, breadcrumb):
             self._modify_nodes(nodes)
         return nodes
 
+
+class AddRatingExtension(Modifier):
+    """Make the RatingExtension available in the menu."""
```

Comment:
I don't think this is necessary???  I had to provide code like this for the icon extension so that a page's icon extension would be available whenever I had a menu entry for the page.  I think the analogy for the rating extension would be if we wanted users to be able to rate any page from its menu entry.